src/frontend/screens/CreateOrJoinScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { View, Text, StyleSheet, Image } from "react-native";
+import Button from "../sharedComponents/Button";
+import { useNavigation } from "react-navigation-hooks";
+import { defineMessages, FormattedMessage } from "react-intl";
+import {
+  NavigationStackOptions,
+  NavigationStackScreenComponent,
+} from "react-navigation-stack";
+
+const m = defineMessages({
+  joinProject: {
+    id: "screens.JoinProjectSplash.joinProject",
+    defaultMessage: "Join a Project",
+  },
+  createProject: {
+    id: "screens.JoinProjectSplash.createProject",
+    defaultMessage: "Create a Project",
+  },
+});
+
+const navOptions: NavigationStackOptions = {
+  headerShown: false,
+};
+
+export const CreateOrJoinScreen: NavigationStackScreenComponent = () => {
+  const { navigate } = useNavigation();
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.logoContainer}>
+        <Image source={require("../images/icon_mapeo_pin.png")} />
+      </View>
+
+      <Text style={styles.title}>Mapeo</Text>
+      <Button
+        style={styles.button}
+        onPress={() => {
+          navigate("JoinProjectQr");
+        }}
+      >
+        <Text style={styles.buttonText}>
+          <FormattedMessage {...m.joinProject} />
+        </Text>
+      </Button>
+      <Button
+        style={styles.button}
+        onPress={() => {
+          navigate("settings");
+        }}
+      >
+        <Text style={styles.buttonText}>
+          <FormattedMessage {...m.createProject} />
+        </Text>
+      </Button>
+    </View>
+  );
+};
+
+//TODO research typesafe option
```

Comment:
is this todo still relevant? pretty sure the `NavigationStackScreenComponent` annotation handles this too

Reply:
Correct, this was an old comment that I just didn't erase